server/main/models/core.py

```diff
@@ -1375,3 +1375,27 @@ def active_in(cls, *, study_id, protocol_id, assay_id=None, id_range=None):
             measurement__assay__protocol_id=protocol_id,
         )
         return active
+
+
+class DefaultUnit(models.Model):
```

Comment:
This model belongs in the `edd.load` module, not in `main`. It's specific to loading/importing data, not the core data models.